### DISCUSSION TITLE
Multiple validations in a single block

### DISCUSSION THREAD

**Discussion by sameershaik1996** [postmanlabs/postman-flows#46] [Ideas]
It would be much cleaner if we can group multiple validations in a single validation block, UI could be something similar to create variables block, where we can add multiple paths and it's corresponding validation

**Comment by samay-v** [comment]
Hey @sameershaik1996,
Are you looking to operate them with logical operators like AND or OR as well, so that the final outcome is still a single boolean (and hence 2 output ports "true" and "false")? Or are you looking to have an output port for each successful validation, more like a switch-case?

**Comment by sameershaik1996** [comment]
Hi @samay-v ,
I am looking to group the validation and produce a single boolean output
